src/se/edu/inclass/Main.py

```diff
@@ -0,0 +1,65 @@
+from data.DataManager import DataManager
+from task.Deadline import Deadline
+from task.Task import Task
+from typing import List
+
+def main():
+    print("Welcome to SimplyDoIt\n")
+    dm = DataManager("./data/data.txt")
+    tasks_data = dm.load_data()
+
+    # Print data
+    print()
+    print("Deadlines before sorting")
+    print_deadlines(tasks_data)
+
+    # Count deadlines
+    print("Total number of deadlines:", count_deadlines(tasks_data))
+
+    # Print deadlines using streams
+    print("\n")
+    print("Deadlines after sorting")
+    print_deadlines_using_stream(tasks_data)
+
+    # Count deadlines using streams
+    print("Total number of deadlines counted using streams:", count_deadlines_using_stream(tasks_data))
+
+    # Filter tasks using streams
+    filtered_list = filter_task_list_using_streams(tasks_data, "Client")
+    print("\nFiltered list of tasks (filtered by keyword \"Client\"):")
+    print_data(filtered_list)
+
+def count_deadlines(tasks_data: List[Task]) -> int:
+    count = 0
+    for task in tasks_data:
+        if isinstance(task, Deadline):
+            count += 1
+    return count
+
+def count_deadlines_using_stream(tasks: List[Task]) -> int:
+    count = sum(1 for task in tasks if isinstance(task, Deadline))
+    return count
+
+def print_data(tasks_data: List[Task]):
+    print("Printing data using iteration")
+    for task in tasks_data:
+        print(task)
+
+def print_deadlines(tasks_data: List[Task]):
+    print("     Printing deadline using iteration")
+    for task in tasks_data:
+        if isinstance(task, Deadline):
+            print(task)
```

Comment:
Can remove one of the prints. Suggest using print by looping for one and print by stream for other

Reply:
okay do you mean we remove the print statements inside the print_deadlines and print_deadlines_using_stream? 

then for the suggestion, we just add comments stating that we printed one with iteration and one with stream?